registry/celo/calldata-Accounts-v1.1.4.2.json

```
@@ -0,0 +1,2376 @@
+        },
+        "authorizeVoteSigner(address,uint8,bytes32,bytes32)": {
+          "$id": "authorizeVoteSigner",
+          "intent": "Authorize Vote Signer",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Signer",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "wallet",
+                  "eoa",
+                  "contract"
+                ],
+                "sources": [
+                  "ens",
+                  "local"
+                ]
+              },
+              "path": "#.signer",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "V - recovery id of ECDSA",
+              "format": "unit",
+              "params": {
+                "base": null,
+                "decimals": null,
+                "prefix": null
+              },
+              "path": "#.v",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "R",
+              "format": "raw",
+              "params": null,
+              "path": "#.r",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "S",
+              "format": "raw",
+              "params": null,
+              "path": "#.s",
+              "value": null
+            }
+          ],
+          "required": [
+            "#.signer",
+            "#.v",
+            "#.r",
+            "#.s"
+          ],
+          "excluded": null
+        },
+        "setName(string)": {
+          "$id": "setName",
+          "intent": "Set Name ",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Name",
+              "format": "raw",
+              "params": null,
+              "path": "#.name",
+              "value": null
+            }
+          ],
+          "required": [
+            "#.name"
+          ],
+          "excluded": null
+        },
+        "setPaymentDelegation(address,uint256)": {
+          "$id": "setPaymentDelegation",
+          "intent": "Delegate Payment",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Beneficiary",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "wallet",
+                  "eoa",
+                  "contract"
+                ],
+                "sources": [
+                  "ens",
+                  "local"
+                ]
+              },
+              "path": "#.beneficiary",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "Fraction",
+              "format": "unit",
+              "params": {
+                "base": "%",
+                "decimals": "",
```

This is where the linter is not happy, `decimals` is mandatory, eg:

```suggestion
                "decimals": 0,
```